### PR TITLE
Fixed some content in some guides of PEM 8.1.0

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Fixed some content in some guides of PEM 8.1.0

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**
- [ ] This PR adds new content
- [ X ] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/pem/8.1.0/pem_inst_guide_windows/02_pem_hardware_software_requirements.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ For information about the platforms and versions supported by PEM, visit the Ent
 
 <https://www.enterprisedb.com/services-support/edb-supported-products-and-platforms#pem>
 
-**Please note**: PEM 8.0.1 is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended that EDB products running on these platforms be migrated to a supported platform.
+**Please note**: PEM 8.1.0 is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended that EDB products running on these platforms be migrated to a supported platform.
```
Comment (drothery-edb) [inline product_docs/docs/pem/8.1.0/pem_inst_guide_windows/02_pem_hardware_software_requirements.mdx:23]: We can remove this note.

**Review by drothery-edb — COMMENTED** [review]
One small change to make

File: product_docs/docs/pem/8.1.0/pem_ent_feat/index.mdx [hunk 2]
```diff
@@ -7,7 +7,7 @@ This guide will acquaint you with the tools and wizards that are built into the
 
 This guide is not a comprehensive resource; rather, it is meant to serve as an aid to help you evaluate the tool and bring you up to speed with the basics of how to use the product. For more detailed information about using PEM's functionality, please see the online help made available by the PEM client.
 
-This document uses `Postgres` to mean either the PostgreSQL or EDB Postgres Advanced Server database.
+This document uses `Postgres` to mean either the PostgreSQL or EDB Postgres Advanced Server database or EDB Postgres Extended (formerly known as 2nQPostgres).
```
Comment (abhilasha-narendra) [inline product_docs/docs/pem/8.1.0/pem_ent_feat/index.mdx:10]: formerly known as 2ndQPostgres. Please fix the product name.

**Review by drothery-edb — APPROVED** [review]
Please incorporate the comments in your next PR. Thanks.

**Pull request merged**